device/globals/item_background.gd

```diff
@@ -0,0 +1,349 @@
+tool
+
+extends "res://globals/interactive.gd"
+
+export var tooltip = ""
+export var action = ""
+export(String,FILE) var events_path = ""
+export var global_id = ""
+export var use_combine = false
+export var inventory = false
+export var use_action_menu = true
+export(int, -1, 360) var interact_angle = -1
+export var talk_animation = "talk"
+export var active = true setget set_active,get_active
+export var placeholders = {}
+export var use_custom_z = false
+
+var anim_notify = null
+var anim_scale_override = null
+
+var ui_anim = null
+
+var event_table = {}
+
+#func _set(name, val):
```

Comment:
Well, that code is duplicated from global/item.gd. Plain copy/paste, so I didn't feel like modifying. 
However, it could be interesting to make some code cleaning everywhere.